checks/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Alerts & Check-Up'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+import * as checkUtil from '../util/checks'
+
+export const CheckLinks = () => {
+  const checkTitles = checkUtil.CHECK_TITLES;
+  return (
+    <ul>
+      {Object.entries(checkTitles).map(([checkGroup, items]) => {
+        return (
+          <li key={checkGroup}>
+            <strong>{checkGroup}</strong>
+            <ul>
+              {Object.entries(items).map(([checkName, checkTitle]) => {
+                return (
+                  <li key={checkName}>
+                    <Link to={`/docs/checks/${checkGroup}/${checkName}`}>
+                      {checkTitle}
+                    </Link>
+                  </li>
+                )
+              })}
+            </ul>
+          </li>
+        )
+      })}
+    </ul>
+  )
+}
+
+To help you discover potential problems, pganalyze automatically performs a number
+of checks on your database.
+
+Checks run regularly, at different frequencies depending on the check type. When
+a problem is detected, an issue is created for the server or database where the
+problem is found.
+
+Issues may have severity "info", "warning", or "critical". Critical issues may
+cause a serious performance problem or lead to a database outage within a matter
```

Comment:
Maybe better if worded like this:

"Critical issues may indicate a serious performance problem or potential database outage within a matter of hours."

(i.e. to separate the issue in pganalyze from the actual problem, since e.g. if you're getting a lot of active query alerts that itself may not be the root cause)

Reply:
Great call.